src/sentry/south_migrations/0437_auto__add_field_projectdebugfile_data__del_unique_projectdebugfile_pro.py

```diff
@@ -0,0 +1,1250 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
```

Comment:
This needs to be dangerous. Not sure if we have a better way now to do this now. In the past we used to opt out of transactional ddl for postgres and issue a create index concurrently manually.

Reply:
The dangerous part of the migration has been moved to #10164.